package.json

```diff
@@ -28,7 +28,7 @@
   "author": "Brigade Engineering",
   "license": "MIT",
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0"
```

Comment:
I think it is unlikely that we will break React 14 before React 16 is out, and even then we are probably pretty safe. I guess my preference would be to continue to support it at least until 16 is out or we have a compelling reason to remove support.